### PR TITLE
[@types/stripe-v3] accept BankAccountOptions on createToken

### PR DESCRIPTION
**Pull request by mastacheata**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ x Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://stripe.com/docs/api/tokens/create_bank_account / https://stripe.com/docs/api/tokens/create_card?lang=node / https://github.com/stripe/stripe-node/blob/e766d366bc070a80498ff14e8cd508ba0abb40b1/types/2019-12-03/Tokens.d.ts#L81

Basically I couldn't find any definition for the string as first parameter version of createToken, but the type of options basically depends on whatever type the Element is.

### CODE REVIEW

**Review by amritk — APPROVED** [review]
Tests pass, looks good
